File: vectori-caracteristici-si-de-frecventa/sursa2.py
prezent = [0] * 10
n = int(input())
while (n):
    cifra = n % 10
    n = n // 10
    if cifra == 2:
        prezent[2] = prezent[2] + 1
if prezent[2] > 0:
    print("cifra 2 apare in numar de " + str(prezent[2]) + " ori")
else:
    print("cifra 2 nu apare in numar")

File: vectori-caracteristici-si-de-frecventa/sursa2.cpp
#include <iostream>
using namespace std;
int main() {
    int prezent[10];
    int n;
    cin >> n;
    while (n) {
        int cifra = n % 10;
        n = n / 10;
        if (cifra == 2)
            prezent[2]++;
    }
    if (prezent[2] > 0)
        cout << "cifra 2 apare in numar de " << prezent[2] << " ori";
    else
        cout << "cifra 2 nu apare in numar";
    return 0;
}